CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+# Contributing
+
+This project is licensed under the [Apache 2.0 license](LICENSE) and accept
+contributions via GitHub pull requests. This document outlines some of the
+conventions on development workflow, commit message formatting, contact points
+and other resources to make it easier to get your contribution accepted.
+
+To maintain a safe and welcoming community, all participants must adhere to the
+project's [Code of Conduct](code-of-conduct.md).
+
+## Certificate of Origin
+
+By contributing to this project you agree to the Developer Certificate of
+Origin (DCO). This document was created by the Linux Kernel community and is a
+simple statement that you, as a contributor, have the legal right to make the
+contribution. See the [DCO](DCO) file for details.
+
+## Email and Chat
+
+The project currently uses the [Kubernetes Slack](https://slack.k8s.io/):
+- [#prometheus-operator](https://kubernetes.slack.com/archives/CFFDS2Z7F)
+- [#prometheus-operator-dev](https://kubernetes.slack.com/archives/C01B03QCSMN)
+
+Please avoid emailing maintainers directly.
+
+## Getting Started
+
+- Fork the repository on GitHub
+- Read the [README](README.md) for build and test instructions
+- Play with the project, submit bugs, submit patches!
+
+## Contribution Flow
+
+This is a rough outline of what a contributor's workflow looks like:
+
+- Create a topic branch from where you want to base your work (usually `main`).
+- Make commits of logical units.
+- Make sure your commit messages are in the proper format (see below).
+- Push your changes to a topic branch in your fork of the repository.
+- Make sure the tests pass, and add any new tests as appropriate.
+- Submit a pull request to the original repository.
```

Comment:
Maybe it is worth mentioning how to execute tests? Just mentioning the Makefile step should be enough I think

Reply:
I don't want to sound like a sellout of my employer, but the `developer-workspace` folder was created as an alternative for contributors who don't have a good local environment set up. Do you think it is worth mentioning it?

Reply:
Yes, totally! Would like to write it in a follow-up PR? I think you have the most knowledge of what should be put here so people use `developer-workspace` and gtipod :)

Reply:
yep, sounds good to me!